Swift/BRCrypto/common/BRCryptoKey.swift

```diff
@@ -29,6 +37,21 @@ public final class Key {
             .map { Key (core: $0)}
     }
 
+    ///
+    /// Create `Key` from `string` using the passphrase to decrypt it. The string must be BIP38
+    /// format. Different crypto currencies have different implementations; this function will
+    /// look for a valid string using BITCOIN mainnet and BITCOIN testnet.
+    ///
+    /// - Parameter string
+    /// - Parameter passphrase
+    ///
+    /// - Returns: A Key if one exists
+    ///
+    static public func createFromString (asPrivate string: String, withPassphrase: String) -> Key? {
```

Comment:
Given that BIP38 defines encrypting private keys, this function should be analogous to `createFromString:asPrivate` - and it is.